firstvoices/backend/models/song.py

```diff
@@ -0,0 +1,54 @@
+from django.db import models
+
+from backend.permissions import predicates
+from .base import AudienceMixin, BaseControlledSiteContentModel
+from .media import RelatedMediaMixin
+from .mixins import AuthouredMixin, dynamic_translated_field_mixin_factory
+
+
+class Song(
+    AuthouredMixin,
+    AudienceMixin,
+    RelatedMediaMixin,
+    dynamic_translated_field_mixin_factory("Song",
+                                           "title",
+                                           unique=False,
+                                           blank=False,
+                                           nullable=False
+                                           ),
```

Comment:
I think it is super useful to have this factory for things like song lyrics and story page content, but not all our translated fields need to be a long text field. The song title should be a CharField, probably with a max length, so I think the song title and translation could just be CharFields right on the model. 

@gmcauliffe and @caforbes Does that make sense to you, or do we need to put English song titles in their own model and include a language code? I have trouble imagining that we'd need multiple translations of a song title

Reply:
CharFields should do the trick, `title` and `titleTranslation`, `intro` and `introTranslation`. Thanks! 👍

Reply:
Thanks, Guy!

So... it sounds like all of these fields actually only need to support a single translation each (title, introduction, lyric). And from Guy's comment below, a song needs to support multiple lyrics.